{"name": "Ручка /api/v1/orders", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 409\n +  where 200 = <Response [200]>.status_code", "trace": "self = <test_api.tests.test_list_orders.TestListOrders object at 0x000001DAFB417C70>\n\n    @allure.title(\"Ручка /api/v1/orders\")\n    @allure.description(\"Получаем список заказов и выводим значение, больше максимально требуемого\")\n    def test_get_list_orders_limit_50_error(self):\n        responce = requests.get(URLS.URL_MAIN + URLS.URL_TAKE_LIST_ORDERS + \"?limit=50&page=0\")\n>       assert responce.status_code == 409\nE       assert 200 == 409\nE        +  where 200 = <Response [200]>.status_code\n\ntest_api\\tests\\test_list_orders.py:29: AssertionError"}, "description": "Получаем список заказов и выводим значение, больше максимально требуемого", "start": 1710612796036, "stop": 1710612813436, "uuid": "76049aff-cce3-46df-9bf9-dfee9ae9fac2", "historyId": "f21e5e537ab1d704092b3bf498a60233", "testCaseId": "f21e5e537ab1d704092b3bf498a60233", "fullName": "test_api.tests.test_list_orders.TestListOrders#test_get_list_orders_limit_50_error", "labels": [{"name": "parentSuite", "value": "test_api.tests"}, {"name": "suite", "value": "test_list_orders"}, {"name": "subSuite", "value": "TestListOrders"}, {"name": "host", "value": "LAPTOP-KS9V29SF"}, {"name": "thread", "value": "24912-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.tests.test_list_orders"}]}